pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -171,6 +172,7 @@ func (m *ManagerImpl) Allocate(resourceName string, devs []string) (*pluginapi.A
 // Register registers a device plugin.
 func (m *ManagerImpl) Register(ctx context.Context, r *pluginapi.RegisterRequest) (*pluginapi.Empty, error) {
 	glog.Infof("Got registration request from device plugin with resource name %q", r.ResourceName)
+	metrics.DevicePluginRegistrationCount.WithLabelValues(r.ResourceName).Inc()
```

Comment:
Just wondering why not count the `successful` registration？

Reply:
I think it is also useful to track failed ones. For now, we can start with a single metric to track both. If needs come, we can consider to add separate metrics for failed and successful registrations.

Reply:
lgtm